docs/velocity/admin/reference/configuration.md

```diff
@@ -37,13 +37,18 @@ These settings mostly cover the basic, most essential settings of the proxy.
 | `bind`                        | Address | This tells the proxy to accept connections on a specific IP. By default, Velocity will listen for connections on all IP addresses on the computer on port 25577.                                                 |
 | `motd`                        | Chat    | This allows you to change the message shown to players when they add your server to their server list. You can use [MiniMessage format](https://docs.advntr.dev/minimessage/format.html).               |
 | `show-max-players`            | Integer | This allows you to customize the number of "maximum" players in the player's server list. Note that Velocity doesn't have a maximum number of players it supports.                                               |
+| `online-mode`                 | Boolean | Should we authenticate players with Mojang? By default, this is on.                                                                                                                                              |
+| `force-key-authentication`    | Boolean | Should the proxy enforce the new public key security standard? By default, this is on.                                                                                                                           |
+| `prevent-client-proxy-connections` | Boolean | If client's ISP/AS sent from this proxy is different from the one from Mojang's authentication server, the player is kicked. This disallows some VPN and proxy connections but is a weak form of protection. |
+| `forwarding-secret-file`      | String | If you are using modern or BungeeGuard IP forwarding, configure a file that contains a unique secret here. The file is expected to be UTF-8 encoded and not empty.                                                |
 | `player-info-forwarding-mode` | Enum    | See [Configuring player information forwarding](../getting-started/forwarding.md) for more information.                                                                                                          |
 | `forwarding-secret`           | String  | This setting is used as a secret to ensure that player info forwarded by Velocity comes from your proxy and not from someone pretending to run Velocity. See the "Player info forwarding" section for more info. |
```

Comment:
This [only seems to exist for old configs (1.0)](https://github.com/PaperMC/Velocity/pull/712#issuecomment-1144401976), should we still have this documented here?

Reply:
I removed forwarding-secret option. Is it okay?